ethpm_cli/manifest.py

```diff
@@ -76,6 +75,168 @@ def generate_custom_manifest(project_dir: Path) -> None:
+            f"License found ({license}). Would you like to change it?"
+        )
+
+    if flag:
+        new_license = input("Enter your new license: ")
+        return b.license(new_license)
+    return None
+
+
+def amend_authors(manifest: Manifest) -> Optional[Callable[..., Manifest]]:
+    try:
+        authors = manifest["meta"]["authors"]
+    except KeyError:
+        flag = parse_bool_flag("No authors found, would you like to add any?")
+    else:
+        flag = parse_bool_flag(
+            f"Authors found ({authors}). Would you like to change them?"
+        )
+
+    if flag:
+        new_authors = input("Enter an author or multiple authors separated by commas: ")
+        return b.authors(*[author.strip() for author in new_authors.split(",")])
+    return None
+
+
+def amend_keywords(manifest: Manifest) -> Optional[Callable[..., Manifest]]:
+    try:
+        keywords = manifest["meta"]["keywords"]
+    except KeyError:
+        flag = parse_bool_flag("No keywords found, would you like to add any?")
+    else:
+        flag = parse_bool_flag(
+            f"Keywords found ({keywords}). Would you like to change them?"
+        )
+
+    if flag:
+        new_keywords = input(
+            "Enter a keyword or multiple keywords separated by commas: "
+        )
+        return b.keywords(*[keyword.strip() for keyword in new_keywords.split(",")])
+    return None
+
+
+def amend_links(manifest: Manifest) -> Optional[Callable[..., Manifest]]:
+    try:
+        links = manifest["meta"]["links"]
+    except KeyError:
+        flag = parse_bool_flag("No links found, would you like to add any?")
+    else:
+        flag = parse_bool_flag(f"Links found ({links}). Would you like to change them?")
+
+    if flag:
+        documentation = input(
+            "Enter a new link for your documentation (leave blank to skip): "
+        )
+        repo = input("Enter a new link for your repository (leave blank to skip): ")
+        website = input("Enter a new link for your website (leave blank to skip): ")
+        link_kwargs = {"documentation": documentation, "repo": repo, "website": website}
+        actual_kwargs = {k: v for k, v in link_kwargs.items() if v}
+        return b.links(**actual_kwargs)
+    return None
+
+
+def amend_deployments(manifest: Manifest) -> Iterable[Callable[..., Manifest]]:
+    try:
+        manifest["deployments"]
+    except KeyError:
+        flag = parse_bool_flag("No deployments found, would you like to add one?")
+    else:
+        # todo: support amending existing deployments, and refactor deployment strategy
+        # to support multiple deployments on the same chain
+        cli_logger.info(
+            f"Deployments found, amending existing deployments is not currently supported."
+        )
+        return tuple()
+
+    if flag:
+        deployment_data = amend_single_deployment(manifest)
+        return (b.deployment(**dep) for dep in deployment_data)
+    return tuple()
+
+
+@to_tuple
```

Comment:
This looks wrong.  The `yield` below returns a dict, and then I'm pretty sure the `to_tuple` is going to convert that to be only the dict keys in a tuple.

Suggests that there may be missing test coverage.

Reply:
It's working as expected - returns `({},)` - though the test coverage could be better, but there's other improvements to the whole deployments strategy i'm planning to add in a future pr

Reply:
I misread what was going on, carry on.